<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Root</class>
 <widget class="QWidget" name="Root">
  <layout class="QHBoxLayout" name="RootLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item>
    <widget class="QSplitter" name="Splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>2</number>
     </property>
     <widget class="FileTreeView" name="FileTree"/>
     <widget class="TargetWidget" name="TargetPages">
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileTreeView</class>
   <extends>QTreeView</extends>
   <header>widgets</header>
  </customwidget>
  <customwidget>
   <class>TargetWidget</class>
   <extends>QTabWidget</extends>
   <header>widgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
